README.md

```diff
@@ -661,6 +661,24 @@ yarn example:android
 Note: You'll probably want to change the Facebook App ID to your own, else the example app won't be able to login.  To change it, edit your local copy of `refresh-example.sh`, update the `FacebookAppId` variable, then re-run `refresh-example.sh` to regenerate the example directory.
 
 
+## Testing with Jest
+
+We have a example mock inside `jest/setup.js` but you just add the following line to your setup file:
+
+```js
+jest.mock('react-native-fbsdk-next', () => require('react-native-fbsdk-next/jest/mocks').default);
```

Comment:
I don't think this will be present in the published package unless this array includes the jest directory ?

https://github.com/thebergamo/react-native-fbsdk-next/blob/e25a711d2d1b3557733cf98ed15ff7ed6b8eecde/package.json#L67-L78

Reply:
done 517860910612865d4f4ea07a2ce1e806fff5cca6